impeller/entity/contents/content_context.cc

```diff
@@ -629,22 +646,6 @@ void ContentContext::InitializeCommonlyUsedShadersIfNeeded() const {
     options.stencil_mode = stencil_mode;
     CreateIfNeeded(clip_pipelines_, options);
   }
-
```

Comment:
We don't need the bootstrap texture anymore since the empty texture warms the same shader cache but is actually useful.

Reply:
Done